docs/openapi_v1.0.0-alpha.yaml

```
@@ -516,20 +575,20 @@ components:
           description: |
             This is the RECOMMENDED refresh interval for the Reseller and SHOULD be used by the Reseller to control the frequency at which they make a `getBooking` request for the following scenarios:
 
-            * To see if a booking/cancellation has changed out of a `PENDING` status into `CONFIRMED` or `REJECTED`.
+            * To see if a booking/cancel has changed out of a `PENDING` status into `CONFIRMED` or `REJECTED`.
```

I think this may have been mistakenly shortened. This refers to either a Booking or Cancellation request.

```suggestion
            * To see if a booking/cancellation has changed out of a `PENDING` status into `CONFIRMED` or `REJECTED`.
```